docs/index.md

```diff
@@ -46,7 +51,10 @@ discussed and shared at the
 
 If you have created your own templates that others might find useful, 
 you are encouraged to share your templates via this repository.
-  
+
+Already available odML templates can be browsed and imported from
```

Comment:
"Already available templates" doesn't work as a sentence starter.  Maybe just "Available templates" or "Existing templates"?

"Currently available templates" would be my choice.